geonode/api/api.py

```diff
@@ -156,6 +157,24 @@ class Meta:
         serializer = CountJSONSerializer()
 
 
+class OwnerResource(TypeFilteredResource):
```

Comment:
I have just sent a pull request in order to reuse the ProfileResource class. 
But I'm not sure if it is better because Profile Resource makes more database queries (by dehydrate functions)